src/extension.ts

```diff
@@ -1,14 +1,32 @@
 import ExtensionCommandHandler from "./extension-command-handler";
 import IPC from "./ipc";
 
-const extensionCommandHandler = new ExtensionCommandHandler();
-const ipc = new IPC(
-  navigator.userAgent.indexOf("Brave") != -1
-    ? "brave"
-    : navigator.userAgent.indexOf("Edg") != -1
-      ? "edge"
-      : "chrome",
-  extensionCommandHandler
-);
+function connect() {
+  const extensionCommandHandler = new ExtensionCommandHandler();
+  const ipc = new IPC(
```

Comment:
hmm, each `IPC` instance has state associated with it, like the `setInterval` calls. I think if we destroy this object, the intervals are going to persist. I think we want to create these objects only once. I think the only thing we need to do when the button is clicked / the alarm is triggered is to wake up the service worker.